chapters/synchronous.tex

```diff
@@ -1120,42 +1120,45 @@ \section{Clocked Discretized Partition}\label{continuous-time-equations-in-clock
-Hereby, a discretized continuous-time partition is seen as a clocked partition.
+Especially, operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate signals of the discretized partition with other partitions.
+Hereby, a discretized partition is treated similarly as a clocked partition.
 
 
-\subsection{Clocked Discrete-Time and Discretized Continuous-Time}\label{clocked-discrete-time-and-clocked-discretized-continuous-time-partition}
+\subsection{Clocked and Discretized Partitions}\label{clocked-discrete-time-and-clocked-discretized-continuous-time-partition}\label{clocked-and-discretized-partition}
 
 Additionally to the variability of expressions defined in \cref{variability-of-expressions}, an orthogonal concept \willintroduce{clocked variability} is defined in this section.
 If not explicitly stated otherwise, an expression with a variability such as \emph{continuous-time} or \emph{discrete-time} means that the expression is inside a partition that is not associated to a clock.
 If an expression is present in a partition that is not a continuous-time partition, it is a \firstuse[clocked!expression]{clocked expression}\index{expression!clocked} and has \firstuse[clocked!variability]{clocked variability}\index{expression variability!clocked}.
 
-After sub-clock inferencing, see \cref{sub-clock-inferencing}, every partition that is associated to a clock has to be categorized as \willintroduce{clocked discrete-time} or \willintroduce{clocked discretized continuous-time} partition.
+After sub-clock inferencing, see \cref{sub-clock-inferencing}, every partition that is associated to a clock has to be categorized as a \willintroduce{clocked} or a \willintroduce{discretized} partition.
+\begin{nonnormative}
+Previously, clocked partition was refered to as \emph{clocked discrete-time partition}, and discretized partition as \emph{clocked discretized continuous-time partition}.
+\end{nonnormative}
 
-If a clocked partition contains no operator \lstinline!der!, \lstinline!delay!, \lstinline!spatialDistribution!, no event related operators from \cref{event-related-operators-with-function-syntax} (with exception of \lstinline!noEvent! and \lstinline!smooth!), and no \lstinline!when!-clause with a \lstinline!Boolean! condition, it is a \firstuse[clocked!discrete-time partition]{clocked discrete-time}\index{partition!clocked discrete-time} partition.
+If a partition that is not continuous-time contains no operator \lstinline!der!, \lstinline!delay!, \lstinline!spatialDistribution!, no event related operators from \cref{event-related-operators-with-function-syntax} (with exception of \lstinline!noEvent! and \lstinline!smooth!), and no \lstinline!when!-clause with a \lstinline!Boolean! condition, it is a \firstuse[clocked!partition]{clocked}\index{partition!clocked} partition.
```

Comment:
Avoiding a negated condition would make it simpler:
```suggestion
If a partition that is not continuous-time contains any of the operators \lstinline!der!, \lstinline!delay!, \lstinline!spatialDistribution!, or event related operators from \cref{event-related-operators-with-function-syntax} (with exception of \lstinline!noEvent! and \lstinline!smooth!), or contains a \lstinline!when!-clause with a \lstinline!Boolean! condition, it is a \firstuse[discretized!partition]{discretized}\index{partition!discretized} partition.
Otherwise, it is a \firstuse[clocked!partition]{clocked}\index{partition!clocked} partition.

\begin{nonnormative}
That is, a clocked partition is a standard sampled data system described by difference equations.
\end{nonnormative}

A discretized partition has to be solved with a \emph{solver method} of \cref{solver-methods}.
```

Note that the suggestion replaces more text than I was able include in the comment's range of lines.